github-config.yaml

```diff
@@ -115,6 +115,16 @@ orgs:
         privacy: closed
         repos:
           community: admin
+      Scrum:
+        description: SCRUM masters
+        maintainers:
+          - durandom
+          - goern
+          - schwesig
+        members:
+        privacy: closed
+        repos:
+          scrum: triage
```

Comment:
And the team needs more than *triage* permissions to e.g. be able to edit existing issues.

Our usage of the `scrum` repo is not the typical dev repo usage, and editing each other's issues would be expected.